tests/e2e/cluster_downgrade_test.go

```diff
@@ -45,15 +48,19 @@ func testDowngradeUpgrade(t *testing.T, clusterSize int) {
 	}
 	currentVersion := semver.New(version.Version)
 	lastVersion := semver.Version{Major: currentVersion.Major, Minor: currentVersion.Minor - 1}
-	currentVersionStr := fmt.Sprintf("%d.%d", currentVersion.Major, currentVersion.Minor)
-	lastVersionStr := fmt.Sprintf("%d.%d", lastVersion.Major, lastVersion.Minor)
+	currentVersionStr := fmt.Sprintf("%d.%d.%d", currentVersion.Major, currentVersion.Minor, currentVersion.Patch)
```

Comment:
Isn't that just a currentVersion.String() now?

Reply:
The cluster version is like `v3.6.0` instead of etcd server version `v3.6.0-alpha0`.
So I go in this way. 😂

Reply:
could we then remove internal version `-alpha.0` from `currentVersion` instead?

Reply:
Updated. PTAL